custom_components/alexa_media/__init__.py

```
@@ -1156,49 +1443,545 @@ def _build_last_called_payload(
     ) -> dict | None:
         """Validate an alexapy history record and return an AMP last_called payload."""
         if not isinstance(last, dict):
+            if debug:
```

Pylint (4.0.5)

[error] Undefined variable 'debug'

(E0602)